libdnf5/base/transaction.cpp

```diff
@@ -1098,7 +1100,11 @@ bool Transaction::Impl::check_gpg_signatures() {
         }
     }
     if (num_checks_skipped > 0) {
-        auto warning_msg = utils::sformat(_("Warning: skipped PGP checks for {} package(s)."), num_checks_skipped);
+        auto repo_string = libdnf5::utils::string::join(repo_of_checks_skipped, ", ");
+        auto warning_msg = utils::sformat(
+            _("Warning: skipped PGP checks for {0} package(s) from {1} repository(s)."),
+            num_checks_skipped,
+            repo_string);
```

Comment:
The "(s)" for "repository" doesn't work as well as for "package". Firstly, it doesn't match the plural form, secondly, it's less clear when there is a list of repositories and not just a number. For example, this reads a bit weird: "skipped PGP checks for 82 package(s) from fedora, updates repository(s)." I would change it, so that the list of repos is at the end: "skipped PGP checks for 82 package(s) from repositories: fedora, updates". What do you think?

Reply:
We can even leverage plural forms here:
```P_("skipped PGP checks for {} package from repository {}", "skipped PGP check for {} packages from repositories: {}", num_checks_skipped)```
Or something similar, plurals translations are currently not used in dnf5 so this would be a pioneer :)

Reply:
I was thinking about `P_`, but there are 3 possibility
"skipped PGP checks for {} package from repository {}"
"skipped PGP checks for {} packages from repository {}"
"skipped PGP checks for {} packages from repositories {}"

Reply:
Yeah, this is not exactly a straightforward example for `P_` as it contains two plurals. The only way would be splitting the message into two parts - one for package(s) and the other for repository(ies).
I'm not sure if this complication for translators is worth it, as in some languages it might be difficult to join such split sentences back together.

Reply:
~~~~
utils::sformat(
  P_(
    "skipped PGP checks for {} package from repositories: {}",
    "skipped PGP check for {} packages from repositories: {}",
     num_checks_skipped),
  num_checks_skipped
  repo_string);
~~~~